dev-python/urwidtrees/urwidtrees-1.0.2-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 python3{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Tree widgets for urwid"
+HOMEPAGE="https://github.com/pazz/urwidtrees"
+SRC_URI="https://github.com/pazz/urwidtrees/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )"
+RDEPEND=">=dev-python/urwid-1.1.0[${PYTHON_USEDEP}]"
+
+src_prepare() {
+	find "${S}" -name '*.py' -print0 | xargs -0 -- sed \
```

Comment:
Could you switch this to `find ... -exec`?

Reply:
Ok fixed